source/game/scene/GameScene.cc

```diff
@@ -40,9 +40,18 @@ void GameScene::reinit() {
 
 void GameScene::appendResource(System::MultiDvdArchive *archive, s32 id) {
     Resource *resource = new Resource(archive, id);
+
+    if (!s_listInitialized) {
+        m_resources.init(offsetof(Resource, m_link));
+        s_listInitialized = true;
+    }
```

Comment:
See 8051a34c. This does not need to be static, nor does it need a boolean to determine whether or not it's initialized.